cove_360/tests_functional.py

```diff
@@ -275,10 +275,10 @@ def test_index_page_360(server_url, browser):
   
 @pytest.mark.parametrize(('link_text', 'url'), [
     ('360Giving Data Standard guidance', 'http://www.threesixtygiving.org/standard/'),
-    ('Excel', 'https://github.com/ThreeSixtyGiving/standard/raw/master/schema/summary-table/360-giving-schema-titles.xlsx'),
-    ('CSV', 'https://github.com/ThreeSixtyGiving/standard/raw/master/schema/summary-table/360-giving-schema-titles.csv/grants.csv'),
+    ('Excel', 'https://threesixtygiving-standard.readthedocs.io/en/latest/_static/summary-table/360-giving-schema-titles.xlsx'),
+    ('CSV', 'https://threesixtygiving-standard.readthedocs.io/en/latest/templates-csv'),
```

Comment:
Just checking you mean this one as its not linking to the grants.csv like before?